app/controllers/assessor_interface/review_verifications_controller.rb

```diff
@@ -11,6 +11,14 @@ def index
       @professional_standing_request =
         assessment.professional_standing_request if assessment.professional_standing_request.verify_failed?
 
+      @region = @assessment.application_form.region
+
+      @teaching_authority_name =
+        (
+          @region.teaching_authority_name.presence ||
+            "Contact relevant competent authority"
+        )
```

Comment:
I think we should use this helper interface: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/29cbfd24eac87c84b5db1e4f9b363e6edfe9f254/app/helpers/region_helper.rb#L14-L16

If we need a custom default value, I think it makes sense to pass that as a new argument to the helper, but I think every region will have a teaching authority name set.